Client/Frontend/Home/RecentTabsPanel.swift

```diff
@@ -0,0 +1,162 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import UIKit
+
+import Shared
+import Storage
+import XCGLogger
+import Deferred
+
+private let log = Logger.browserLogger
+
+class RecentTabsPanel: UIViewController, HomePanel {
+    weak var homePanelDelegate: HomePanelDelegate? = nil
+    var profile: Profile!
+    weak var historyPanel: HistoryPanel? = nil
+    private lazy var recentlyClosedHeader: UILabel = {
+        let headerLabel = UILabel()
+        headerLabel.text = Strings.RecentTabsPanelTitle
+        headerLabel.font = DynamicFontHelper.defaultHelper.DeviceFontHistoryPanel
+        headerLabel.textAlignment = .Center
+        headerLabel.backgroundColor = UIColor.whiteColor()
+        return headerLabel
+    }()
+    private lazy var tableViewController: RecentTabsPanelSiteTableViewController = {
```

Comment:
Nit: No need for the closure - you can just do `RecentTabsPanelSiteTableViewController()` and it will still be invoked lazily.